src/APIs/trivia-facts.ts

```diff
@@ -0,0 +1,14 @@
+export const triviaFacts = [
+  {
+    fact: 'Brad is awesome',
+    source: 'nhcarrigan'
+  },
+  {
+    fact: 'nhcarrigan is awesome',
+    source: 'Everyone!'
+  },
+  {
+    fact: 'bjorno might be evil',
+    source: 'bjorno'
+  }
+];
```

Comment:
Do you think we should make this a JSON file, rather than TS file. Its still up in the air of this is a good idea or not, but having it as JSON would make this a good target as an easy "beginner's" PR sorta file/feature.

Reply:
That's a good idea. I'll refactor that tomorrow. 🙂